test/e2e/flask/multichain-api/wallet_notify.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { strict as assert } from 'assert';
+import { withFixtures } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import {
+  DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+  type FixtureCallbackArgs,
+  openMultichainDappAndConnectWalletWithExternallyConnectable,
+} from './testHelpers';
+
+describe('Calling `eth_subscribe` on a particular network event', function () {
+  it('Should receive a notification through the Multichain API for the event app subscribed to', async function () {
+    await withFixtures(
+      {
+        title: this.test?.fullTitle(),
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToMultichainTestDapp()
+          .build(),
+        ...DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+      },
+      async ({ driver, extensionId }: FixtureCallbackArgs) => {
+        await openMultichainDappAndConnectWalletWithExternallyConnectable(
+          driver,
+          extensionId,
+        );
+        const SCOPE = 'eip155:1337';
+
+        await driver.clickElementSafe(
+          `[data-testid="${SCOPE}-eth_subscribe-option"]`,
+        );
+        await driver.clickElementSafe(
+          `[data-testid="invoke-method-${SCOPE}-btn"]`,
+        );
+
+        const walletNotifyNotificationWebElement = await driver.findElement(
+          '#wallet-notify-result-0',
+        );
+        const resultSummaries = await driver.findElements('.result-summary');
+
+        /**
+         * Currently we don't have `data-testid` setup for the desired result, so we click on all available results
```

Comment:
Seems less than ideal, perhaps worth it to address 😄

Reply:
Agreed. We have a ticket on our board to improve the multichain test dapp to add `data-testids`: https://github.com/orgs/MetaMask/projects/146/views/1?filterQuery=label%3A%22team-wallet-api-platform%22+-status%3ABacklog&pane=issue&itemId=93881732&issue=MetaMask%7Ctest-dapp-multichain%7C45